packages/ui-components/src/components/ComboBox/ComboBox.component.tsx

```diff
@@ -386,7 +408,6 @@ export const ComboBox = ({
 
                 {!hasError && !isLoading ? (
                   <Combobox.Button
-                    disabled={disabled}
                     className={`
                         juno-combobox-toggle 
                         ${buttonStyles}
```

Comment:
**(Suggested Improvement, Not Blocking)**

We could return `null` here to not render anything at all.

Reply:
Done